src/js/components/List.jsx

```diff
@@ -7,6 +9,7 @@ type Props = {
     style: Object,
     data?: Array<Object>,
     labelFunction?: Function,
+    disabled?: bool,
```

Comment:
Not saying to change this because it's fine, just a tip that you might get less curry from Flow if instead of making things like disabled optional, you just add a false / empty value to `defaultProps`